contracts/SpokePool.sol

```diff
@@ -57,7 +57,7 @@ abstract contract SpokePool is
 
     // Address of wrappedNativeToken contract for this network. If an origin token matches this, then the caller can
     // optionally instruct this contract to wrap native tokens when depositing (ie ETH->WETH or MATIC->WMATIC).
-    WETH9Interface public wrappedNativeToken;
+    WETH9Interface public DEPRECATED_wrappedNativeToken;
```

Comment:
upgrading a proxy with this impl. means we can't remove this storage slot but I think we can rename it